frontstage_api/Resources/secure_messaging.py

```diff
@@ -5,22 +5,115 @@
 from structlog import wrap_logger
```

Comment:
Minor point but we should lowercase the Resources directory for consistency

Reply:
done :+1: